tests/auto/ci_auto.py

```diff
@@ -125,8 +127,12 @@ def __init__(self, preq_dict, ghinterface_obj, machine_dict, compiler,
                  repo):
         self.logger = logging.getLogger('JOB')
         self.preq_dict = preq_dict
-        # both build and WE2E tests call same module
-        self.job_mod = importlib.import_module('jobs.build')
+        # both SRWA build and WE2E tests call same module build.py
+        # GSI regression tests call regr.py
+        if self.preq_dict["action"] == "rt":
+            self.job_mod = importlib.import_module('jobs.regr')
+        else:
+            self.job_mod = importlib.import_module('jobs.build')
```

Comment:
As we discussed in today's in-person meeting, you should be able to skip this kind of syntax and adopt the standard:

```
import jobs
...
        if self.preq_dict["action"] == "rt":
            self.job_mod = jobs.regr
        else:
            self.job_mod = jobs.build
```